scanomatic/models/factories/scanning_factory.py

```diff
@@ -434,6 +434,14 @@ def _validate_aux_info(cls, model):
             return True
         return model.FIELD_TYPES.auxillary_info
 
+    @classmethod
+    def all_keys_valid(cls, keys):
```

Comment:
I can't find any calls to this in the code, for this model. Did you forget to commit something or is there something obscure that I'm not seeing?

Reply:
it's called from a method defined on the base class

Reply:
there's a lot of fancy things with this old version of the serialization.

Reply:
look at the load_first method of the serializer.